src/transformers/models/opt/modeling_opt.py

```diff
@@ -71,17 +79,10 @@ def __init__(self, num_embeddings: int, embedding_dim: int):
         self.offset = 2
         super().__init__(num_embeddings + self.offset, embedding_dim)
 
-    def forward(self, attention_mask: torch.LongTensor, past_key_values_length: int = 0):
+    def forward(self, position_ids: torch.LongTensor):
```

Comment:
that is kind of a breaking change for this module 😓

Reply:
why is this a problem? the weights are the same so loading should work. and this module should not be used by outside code so it is not supposed to break anything.

Reply:
Yeah but it has caused issue in the past 😉

Reply:
OK, how do you think I should do it? The module should get position_ids to work with packed sentences. Should I add position_ids as the last argument with None as default?